<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyricShow</class>
 <widget class="QWidget" name="lyricShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_lyric">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>39</y>
     <width>421</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	font-family: &quot;Microsoft YaHei&quot;;
    padding: 3px 6px;
    border-radius: 6px;
	background: #FFFFFF;
	font-size: 16px;
	font-weight: 400;
	line-height: 66px;
	color: #16181A;
}
 
QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 6px;
}
 
QScrollBar::handle:vertical
{
       background-color: rgba(255,255,255,1);
       min-height: 30px;
	  border-radius:3px;
	  width: 6px;
   }
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
 }
 
 QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
 QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
 {	
	background:transparent;
      border-radius:3px;
}</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_songImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>歌曲名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>445</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>专辑名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_songName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 15px;
    color: #BDC8E2;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_albumName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>445</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 15px;
    color: #BDC8E2;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_singerName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 15px;
    color: #BDC8E2;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
